src/transformers/utils/notebook.py

```diff
@@ -341,12 +383,9 @@ def on_evaluate(self, args, state, control, metrics=None, **kwargs):
             _ = metrics.pop(f"{metric_key_prefix}_steps_per_second", None)
             _ = metrics.pop(f"{metric_key_prefix}_jit_compilation_time", None)
             for k, v in metrics.items():
-                if k == f"{metric_key_prefix}_loss":
-                    values["Validation Loss"] = v
```

Comment:
And here

Reply:
Here's a problem of `Validation Loss`. If we have multiple eval datasets, the loss of the last dataset will overwrite previous values. `on_evaluate` is called for each dataset seperately so the function may not know if there are multiple datasets. Got any idea how to overcome this?